### ISSUE TITLE
Target size 2.5.8, CSS pixels and high resolution devices, Native apps, and the "reference pixel"

### ISSUE THREAD

**Issue by anevins12** [w3c/wcag#3730]
I'm a little muddled in my understanding of how to test Target Size Minimum (2.5.8) on native apps with high resolution devices.

Without having the tools to do this natively and not even the Android Accessibility Scanner app is sufficient, I have reverted to figuring this out on desktop. I do this via screenshotting the native app UI in question, transferring that to desktop, and then using an image editor tool to measure pixels.

However, I understand that before any such measurement can take place I need to first establish the true measurement of the image in CSS Pixels.

Where things fall down for me is understanding the "viewing distance."

CSS pixels are defined by WCAG as as unit that is density-independent and should, as close as possible, match the [CSS Values and Units Module Level 3 reference pixel](https://www.w3.org/TR/css3-values/#reference-pixel). Taking into account the physical dimensions of the display and the viewing distance. 
https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-css-pixel

I've not heard my peers talk about viewing distance before and wanted to be sure that this was something I had to think about in my role as an accessibility auditor.

Do I have to calculate viewing distance?

**Comment by mraccess77** [comment]
This is being discussed in the Guidance on Applying WCAG to Non-web ICT task force that is working on updating that document.  At this time draft proposals have a note indicating it would equate it to a "platform-defined density-independent pixel" https://w3c.github.io/wcag2ict/#dfn-css-pixel

**Comment by patrickhlauke** [comment]
> I've not heard my peers talk about viewing distance before and wanted to be sure that this was something I had to think about in my role as an accessibility auditor.

No, that's a consideration the device manufacturers make when they decide what the actual physical size of their platform/OS basic density-independent measure is. For all intents and purposes, when evaluating native apps on Android/iOS, you basically just read "CSS px" as meaning  `dp` for Android, `point` for iOS

**Comment by anevins12** [comment]
That makes a lot more sense on reflection
